web/modules/custom/os2forms_selvbetjening/src/Helper/FormHelper.php

```diff
@@ -143,7 +139,19 @@ public function validateByContentFunction(array &$form, FormStateInterface $form
         return;
       }
       // Get the number of parameters for the defined function.
-      $functionParamCount = (new \ReflectionFunction($functionName))->getNumberOfRequiredParameters();
+      try {
+        $functionParamCount = (new \ReflectionFunction($functionName))->getNumberOfRequiredParameters();
+      }
+      catch (\ReflectionException $e) {
+        $form_state->setError(
+          $form['spv'],
+          $this->t('Unable to reflect function %function_name: @error', [
```

Comment:
Not many users know what “reflecting a function” means, so you could consider telling a white lie (i.e. hide the awful truth a bit) and just say

```suggestion
          $this->t('Invalid function %function_name: @error', [
```

You could also consider not showing the actual error (and maybe log it instead).